src/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/KerrSchild.cpp

```diff
@@ -0,0 +1,172 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/KerrSchild.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace EinsteinSolutions {
+
+KerrSchild::KerrSchild(const double mass, KerrSchild::Spin::type spin,
+                       KerrSchild::Center::type center,
+                       const OptionContext& /* context */) noexcept
+    : mass_(mass),
+      // clang-tidy: do not std::move trivial types.
+      center_(std::move(center)),  // NOLINT
+      // clang-tidy: do not std::move trivial types.
+      spin_(std::move(spin))  // NOLINT
+{
+  const double spin_magnitude = magnitude(spin_);
+  if (spin_magnitude > 1.0) {
```

Comment:
Is it possible for this to determined as a PARSE_ERROR or something along those lines? I'm not sure.

Reply:
I don't know.  This constructor, however, should be callable without using Options.

Reply:
Use PARSE_ERROR.  It will do something reasonable if the context is default constructed.